packages/core/src/components/essentials/legend.ts

```diff
@@ -94,7 +93,47 @@ export class Legend extends Component {
 			addedLegendItemsText.html((d) => d.name);
 		}
 
-		this.breakItemsIntoLines(addedLegendItems);
+		const radiusLabel = Tools.getProperty(
```

Comment:
legend probably doesn't need to know about bubble.

2 ways to fix this:
1. A separate class called `BubbleChartLegend` or something of that sort (extends `Legend`), and in there we add the extra label.
2. Legend component accepts `configs` (similar to `Skeleton` - if you look inside `src/charts/scatter.ts` for isntance) and in there we pass in an "extra legend items" array? I'm assuming we'll have more extra legend items in the future so this idea might work better